e2e/cypress/integration/accessibility/accessibility_account_settings_spec.js

```diff
@@ -11,25 +11,56 @@
-                cy.get(`#${section.key}Title`).should('be.visible').and('contain', section.label);
-                if (index < sections.length - 1) {
-                    cy.focused().tab();
-                }
-            }
-        });
-    });
-}
+import {isMac} from '../../utils';
 
 describe('Verify Accessibility Support in different sections in Account Settings Dialog', () => {
+    const accountSettings = {
+        profile: [
+            {key: 'name', label: 'Full Name', type: 'text'},
+            {key: 'username', label: 'Username', type: 'text'},
+            {key: 'nickname', label: 'Nickname', type: 'text'},
+            {key: 'position', label: 'Position', type: 'text'},
+            {key: 'email', label: 'Email', type: 'text'},
+            {key: 'picture', label: 'Profile Picture', type: 'image'},
+        ],
+        security: [
+            {key: 'password', label: 'Password', type: 'text'},
+            {key: 'mfa', label: 'Multi-factor Authentication', type: 'optional'},
+            {key: 'signin', label: 'Sign-in Method', type: 'optional'},
+        ],
+    };
+
+    const settings = {
+        notifications: [
+            {key: 'desktop', label: 'Desktop Notifications', type: 'radio'},
+            {key: 'email', label: 'Email Notifications', type: 'radio'},
+            {key: 'push', label: 'Mobile Push Notifications', type: 'radio'},
+            {key: 'keys', label: 'Words That Trigger Mentions', type: 'checkbox'},
+            {key: 'comments', label: 'Reply notifications', type: 'radio'},
+        ],
+        display: [
+            {key: 'theme', label: 'Theme', type: 'radio'},
+            {key: 'clock', label: 'Clock Display', type: 'radio'},
+            {key: 'name_format', label: 'Teammate Name Display', type: 'none'},
+            {key: 'availabilityStatus', label: 'Show online availability on profile images', type: 'radio'},
+            {key: 'timezone', label: 'Timezone', type: 'none'},
+            {key: 'collapse', label: 'Default Appearance of Image Previews', type: 'radio'},
+            {key: 'message_display', label: 'Message Display', type: 'radio'},
+            {key: 'channel_display_mode', label: 'Channel Display', type: 'radio'},
+            {key: 'languages', label: 'Language', type: 'dropdown'},
+        ],
+        sidebar: [
+            {key: 'showUnreadsCategory', label: 'Group unread channels separately', type: 'multiple'},
+            {key: 'limitVisibleGMsDMs', label: 'Number of direct messages to show', type: 'radio'},
+        ],
+        advanced: [
+            {key: 'advancedCtrlSend', label: `Send Messages on ${isMac() ? '⌘+ENTER' : 'CTRL+ENTER'}`, type: 'radio'},
+            {key: 'formatting', label: 'Enable Post Formatting', type: 'radio'},
+            {key: 'joinLeave', label: 'Enable Join/Leave Messages', type: 'radio'},
+            {key: 'advancedPreviewFeatures', label: 'Preview Pre-release Features', type: 'checkbox'},
+        ],
+    };
```

Comment:
Move test data here since it's only used by this spec file. Collocated to easily maintain if there will be changes in the future.